content/en/guides/hosting/hosting-options/self-managed/kubernetes-operator/_index.md

```diff
@@ -889,6 +893,322 @@ app:
 
 The same concept applies to `console`, `weave`, `weave-trace` and `parquet`.
 
+### OpenTelemetry (OTEL) exporters
+
+Configure OpenTelemetry exporters to send telemetry data (traces, metrics, and logs) from your W&B deployment to external observability platforms like New Relic, Datadog, or other OTLP-compatible services.
+
+#### Basic configuration
+
+To enable OTEL and configure the OTLP HTTP exporter:
+
+```yaml
+otel:
+  enabled: true
+  daemonset:
+    config:
+      exporters:
+        otlphttp:
+          endpoint: "https://example.com:4318"  # Required - your OTEL collector endpoint
+          headers:
+            api-key: "<API-KEY>"  # Add any required authentication headers
+      service:
+        pipelines:
+          traces:
+            receivers: [otlp]
+            processors: [memory_limiter, batch]
+            exporters: [otlphttp]
+          metrics:
+            receivers: [otlp]
+            processors: [memory_limiter, batch]
+            exporters: [otlphttp]
+          logs:
+            receivers: [otlp]
+            processors: [memory_limiter, batch]
+            exporters: [otlphttp]
+```
+
+#### Use Kubernetes secrets for sensitive data
+
+For production deployments, use Kubernetes secrets to store sensitive information like API keys:
+
+```yaml
+otel:
+  enabled: true
+  daemonset:
+    config:
+      exporters:
+        otlphttp:
+          endpoint: "https://example.com:4318"
+          headers:
+            api-key: "${OTEL_API_KEY}"  # Reference environment variable
+    extraEnvs:
+      - name: OTEL_API_KEY
+        valueFrom:
+          secretKeyRef:
+            name: otel-secrets
+            key: api-key
+```
+
+Create the secret before deploying:
```

Comment:
Format this (including the kubectl command) as an alert using the Hugo shortcode.